### DISCUSSION TITLE
Is there a way I can override the CSS of my chirpy template? I want the change the font style, size of the logo etc etc

### DISCUSSION THREAD

**Discussion by HN026** [cotes2020/jekyll-theme-chirpy#1622] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### What is the topic?

Sharing tips and tricks

### Description

I'm not able to find a way to override the CSS and blogs on the net aren't helping much tbf

**Comment by kungfux** [comment, marked as answer]
You can override styles by adding new ones to `assets/css/jekyll-theme-chirpy.scss`.

Please find more info in [documentation](https://chirpy.cotes.page/posts/getting-started/#customizing-stylesheet) and examples in other [discussions](https://github.com/cotes2020/jekyll-theme-chirpy/discussions/categories/q-a?discussions_q=category%3AQ%26A+css).
